clang/test/Analysis/uninit-vals-union.c

```diff
@@ -8,6 +9,6 @@ typedef struct { int x; } A;
 
 void foo(void) {
   U u = {};
-  A *a = &u; // expected-warning{{incompatible pointer types}}
+  A *a = (A*)&u;
```

Comment:
So I see the `no-crash` below and worry a bit b/c sometimes crashes can be dependent on previous issues on previous lines. So maybe this is not longer a regression test?

Reply:
I suppose we can also use `-Wno-error` for this test; I tried to avoid doing that in case we ever make this a hard error because then all those tests will have to be updated again...